chromium_src/net/base/lookup_string_in_fixed_set.cc

```diff
@@ -41,13 +41,16 @@ int LookupSuffixInReversedSet(const unsigned char* graph,
     return kDafsaFound;
   }
 
-  // Recognize .crypto and .eth as known TLDs for decentralized DNS support.
-  // With this, when users type *.crypto or *.eth in omnibox, it will be parsed
-  // as OmniboxInputType::URL input type instead of OmniboxInputType::UNKNOWN,
-  // The first entry in the autocomplete list will be URL instead of search.
-  if (base::EndsWith(host, decentralized_dns::kCryptoDomain)) {
-    *suffix_length = strlen(decentralized_dns::kCryptoDomain) - 1;
-    return kDafsaFound;
+  // Recognize .crypto(and other ud suffixes) and .eth as known TLDs for
+  // decentralized DNS support. With this, when users type *.crypto or *.eth in
+  // omnibox, it will be parsed as OmniboxInputType::URL input type instead of
+  // OmniboxInputType::UNKNOWN, The first entry in the autocomplete list will be
+  // URL instead of search.
+  for (auto* unstoppabe_domain : decentralized_dns::kUnstoppableDomains) {
```

Comment:
nit: should be `unstoppable_domain` not `unstoppabe_domain`

Reply:
https://github.com/brave/brave-core/pull/13202/commits/4f916563f379813df4f0438dbd77bad3f10553d2